### PR TITLE
fix: how-to-setup-api-in-a-local-instance.mdx in docs with correction after api-v2

### PR DESCRIPTION
**Pull request by sathuli2710**
After the implementation of api-v2, some of the documents are not yet updated and could you make developers spend too much time on finding these things. 

## What does this PR do?

- Fixes #XXXX (GitHub issue number)
- Fixes CAL-XXXX (Linear issue number - should be visible at the bottom of the GitHub issue description)
- This is an issue I noticed while going through the docs.
- This changes some text in the documentation for how to setup api in local instance page

## Mandatory Tasks (DO NOT REMOVE)

- [✅] I have self-reviewed the code (A decent size PR without self-review might be rejected).
- [✅] I have updated the developer docs in /docs if this PR makes changes that would require a [documentation change](https://cal.com/docs). If N/A, write N/A here and check the checkbox.
- [N/A] I confirm automated tests are in place that prove my fix is effective or that my feature works.

## How should this be tested?

- Are there environment variables that should be set? No
- What are the minimal test data to have? NA
- What is expected (happy path) to have (input and output)? Check the edited docs page after update under the link https://cal.com/docs/developing/guides/api/how-to-setup-api-in-a-local-instance
- Any other important info that could help to test that PR

## Checklist

- I haven't checked if my changes generate no new warnings

### CODE REVIEW

File: docs/developing/guides/api/how-to-setup-api-in-a-local-instance.mdx [hunk 1]
```diff
@@ -23,8 +23,8 @@ To test API in your local instance, you have the following pre-requisites:
         Start the cal.com server using `yarn dev` on localhost and create the test API keys by visiting `/settings/developer/api-keys`
     </Step>
     <Step title="Start API server and test API">
-        Start the API server by running `yarn workspace @calcom/api dev` and start testing your API locally
+        Start the API server by running `yarn workspace @calcom/api dev` (or `yarn workspace @calcom/api-v2 dev` if api-v2) and start testing your API locally
```
Comment (Praashh) [inline docs/developing/guides/api/how-to-setup-api-in-a-local-instance.mdx:26]:
```suggestion
        Start the API server by running `yarn workspace @calcom/api dev` and start testing your API locally
```

let's keep it as it is for now.

Comment (Praashh) [reply docs/developing/guides/api/how-to-setup-api-in-a-local-instance.mdx:26]: @sathuli2710 please sign the CLA as well.

**Review by Praashh — APPROVED** [review]

File: docs/developing/guides/api/how-to-setup-api-in-a-local-instance.mdx (hunk 1 above)
Comment (sathuli2710) [reply docs/developing/guides/api/how-to-setup-api-in-a-local-instance.mdx]: Sorry I am new to open source, can you please guide me to sign CLA. Looking forward to contribute more in the future.

Comment (Praashh) [reply docs/developing/guides/api/how-to-setup-api-in-a-local-instance.mdx]: nvm it's merged now.

**Pull request merged**